macos-cursor-controller/swiftForNoddy/ToolBarViewController.swift

```diff
@@ -0,0 +1,154 @@
+import Cocoa
+
+class ToolBarViewController: NSViewController {
+    // cursor mode
+    @IBOutlet weak var cursorSensitivitySlider: NSSlider!
+    @IBOutlet weak var cursorReactionSpeedSlider: NSSlider!
+
+    @IBOutlet weak var toggleModePopUp: NSPopUpButton!
+    @IBOutlet weak var leftClickPopUp: NSPopUpButton!
+    @IBOutlet weak var rightClickPopUp: NSPopUpButton!
+    @IBOutlet weak var pauseControlPopUp: NSPopUpButton!
+    @IBOutlet weak var moveCursorUpPopUp: NSPopUpButton!
+    @IBOutlet weak var moveCursorDownPopUp: NSPopUpButton!
+    @IBOutlet weak var moveCursorLeftPopUp: NSPopUpButton!
+    @IBOutlet weak var moveCursorRightPopUp: NSPopUpButton!
+
+    // scroll mode
+    @IBOutlet weak var scrollSensitivitySlider: NSSlider!
+    @IBOutlet weak var pauseScrollPopUp: NSPopUpButton!
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+
+        //cursor mode
+        setSlider(cursorSensitivitySlider, min: 1, max: 10, value: cursorSensitivity, ticks: 11, bool: true)
+        setSlider(cursorReactionSpeedSlider, min: 0.1, max: 1, value: filterAlpha, ticks: 11, bool: true)
+
+        setPopupButton(toggleModePopUp, selectedItems: "Tab")
+        setPopupButton(leftClickPopUp, selectedItems: "F9")
+        setPopupButton(rightClickPopUp, selectedItems: "F10")
+        setPopupButton(pauseControlPopUp, selectedItems: "F11")
+        setPopupButton(moveCursorUpPopUp, selectedItems: "ArrowUp")
+        setPopupButton(moveCursorDownPopUp, selectedItems: "ArrowDown")
+        setPopupButton(moveCursorLeftPopUp, selectedItems: "ArrowLeft")
+        setPopupButton(moveCursorRightPopUp, selectedItems: "ArrowRight")
+
+        //scroll mode
+        setSlider(scrollSensitivitySlider, min: 50, max: 550, value: scrollSensitivity, ticks: 11, bool: false)
+
+        setPopupButton(pauseScrollPopUp, selectedItems: "Q")
+    }
+
+    // cursor mode
+    @IBAction func cursorSensitivityChanged(_ sender: NSSlider) {
+        cursorSensitivity = sender.doubleValue
+    }
+    @IBAction func cursorReactionSpeedChanged(_ sender: NSSlider) {
+        filterAlpha = sender.doubleValue
+    }
```

Comment:
```suggestion
    @IBAction func changeCursorSensitivity(_ sender: NSSlider) {
        cursorSensitivity = sender.doubleValue
    }
    @IBAction func changeCursorReactionSpeed(_ sender: NSSlider) {
        filterAlpha = sender.doubleValue
    }
```
함수 네임은 일반적으로 어떤 기능에 대한 action을 한다는 느낌을 주어야한다고 생각합니다.
또한, 애플의 [API Design Guidelines](https://www.swift.org/documentation/api-design-guidelines/)에는  함수 이름을 동사로 시작하는 것을 권장한다는 내용이 있습니다.
따라서, 위와 같이 함수 시작을 change나 update 등의 동사를 사용함으로써 해당 함수는 "_어떤 행위를 한다_"는 느낌을 주면 좋을거 같습니다.

Reply:
급하게 작업을 하다 보니 기초적인 부분을 놓친 것 같습니다.
추천해주신 사항은 바로 반영해서 올리도록 하겠습니다.

Reply:
함수의 네이밍은 코드 상에서 해당 함수가 어떤 기능을 수행하는지 일차적으로 보여줄 수 있는 부분이라고 생각합니다.
반영해주셔서 감사합니다!

Reply:
형윤님 말씀에 동의합니다. 함수명은 코드를 처음 접하는 사람도 해당 함수의 역할을 직관적으로 이해할 수 있도록 돕는 중요한 요소라고 생각합니다. 코드 리뷰 과정에서 불편을 드린 점 사과드리며, 앞으로 더 신중하게 작성하도록 하겠습니다. 감사합니다!